data/json/recipes/other/tools.json

```
@@ -131,9 +131,9 @@
     "category": "CC_OTHER",
     "subcategory": "CSC_OTHER_TOOLS",
     "skill_used": "fabrication",
-    "skills_required": [ "gun", 3 ],
     "difficulty": 4,
     "time": "20 m",
+    "autolearn": [ [ "fabrication", 6 ] ],
     "book_learn": [ [ "recipe_bullets", 6 ], [ "textbook_fabrication", 7 ] ],
```

You'll want the `book_learn` levels to be lower than the autolearn level, so presumably 4/5 instead of 6/7 if you're setting autolearn to level 6.
```suggestion
    "book_learn": [ [ "recipe_bullets", 4 ], [ "textbook_fabrication", 5 ] ],
```